docs/contribute/guide_book.md

```diff
@@ -0,0 +1,236 @@
+## Overview
```

Comment:
Consider raising the heading level throughout, e.g. changing `##` to `#`, `###` to `##` etc.